app.py

```diff
@@ -234,6 +234,130 @@ def cleanup_html_data(html_data):
     return (filedata, filedata2, filedata3)
 
 
+def readability_score(final_page):
+    # calculate readability score
+    html = final_page
+    original_soup = BeautifulSoup(html, features="lxml").find("main")
+    original_text = original_soup.get_text()
+    original_text = original_text.replace("..", ".")
+    original_text = original_text.replace(".", ". ")
+    original_text = original_text[: original_text.find("defPreFooter")]
+    original_text = original_text.replace("\n", "")
+    original_text = original_text.replace("\t", "")
+    original_text = original_text.replace("\r", "")
+
+    # get initial readability total_score
+    import textstat
+
+    original_score = textstat.flesch_kincaid_grade(original_text)
+    original_score = format(original_score, ".2f")
+
+    # add periods after bullet points and headings so that the Flesch Kicaid score considers them as sentences
+    html1 = html.replace("</li>", ".</li>")
+    html2 = html1.replace("</h1>", ".</h1>")
+    html3 = html2.replace("</h2>", ".</h2>")
+    html4 = html3.replace("</h3>", ".</h3>")
+    html5 = html4.replace("</h4>", ".</h4>")
+    html6 = html5.replace("</h5>", ".</h5>")
+    html7 = html6.replace("</h6>", ".</h6>")
+
+    # get adjusted readability total_score
+    revised_soup = BeautifulSoup(html7, features="lxml").find("main")
+    for t in revised_soup.select("table"):
+        t.extract()
+    revised_text = revised_soup.get_text()
+    revised_text = revised_text.replace("..", ".")
+    revised_text = revised_text.replace(".", ". ")
+    revised_text = revised_text[: revised_text.find("defPreFooter")]
+    revised_text = revised_text.replace("\n", "")
+    revised_text = revised_text.replace("\t", "")
+    revised_text = revised_text.replace("\r", "")
+
+    final_fk = textstat.flesch_kincaid_grade(revised_text)
+
+    from nltk.tokenize import RegexpTokenizer
+
+    tokenizer = RegexpTokenizer('\s+', gaps=True)
```

Comment:
This should be fixed pay declaring the regex as a raw string, see here: https://stackoverflow.com/questions/50504500/deprecationwarning-invalid-escape-sequence-what-to-use-instead-of-d

Replace with 

```python
tokenizer = RegexpTokenizer(r'\s+', gaps=True)
```
And I think it should pass the invalid escape sequence. 

See https://www.flake8rules.com/rules/W605.html